src/pybind/mgr/cephadm/module.py

```diff
@@ -1176,8 +1176,8 @@ def _hosts_with_daemon_inventory(self) -> List[HostSpec]:
         """
         return [
             h for h in self.inventory.all_specs()
-            if self.cache.host_had_daemon_refresh(h.hostname) and
-            h.status.lower() not in ['maintenance', 'offline']
+            if self.cache.host_had_daemon_refresh(h.hostname)
+            and h.status.lower() not in ['maintenance', 'offline']
```

Comment:
the in indentation is wrong. Line 1180 should be indented as we're in the `if` block

Reply:
hmm it looks wrong to me too, but `autopep8 --select W504` insists this is the correct formatting.

likewise adding the extra indention causes this:
```
cephadm/module.py:1180:17: E131 continuation line unaligned for hanging indent
```

Reply:
maybe we refactor this block?

```diff
diff --git a/src/pybind/mgr/cephadm/module.py b/src/pybind/mgr/cephadm/module.py
index 7be272daff7..1283a6ae3c9 100644
--- a/src/pybind/mgr/cephadm/module.py
+++ b/src/pybind/mgr/cephadm/module.py
@@ -1174,11 +1174,12 @@ class CephadmOrchestrator(orchestrator.Orchestrator, MgrModule,
         ``_apply_all_specs()`` was called. thus we end up with a hosts
         where daemons might be running, but we have not yet detected them.
         """
-        return [
-            h for h in self.inventory.all_specs()
-            if self.cache.host_had_daemon_refresh(h.hostname) and
-            h.status.lower() not in ['maintenance', 'offline']
-        ]
+        host_specs: List[HostSpec] = []
+        for h in self.inventory.all_specs():
+            if self.cache.host_had_daemon_refresh(h.hostname) \
+                    and h.status.lower() not in ['maintenance', 'offline']:
+                host_specs.append(h)
+        return host_specs
```

... but not sure that's any more readable. wdyt?